nixos/modules/system/boot/systemd.nix

```diff
@@ -406,20 +407,16 @@ in
       '';
     };
 
-    enableCgroupAccounting = mkOption {
-      default = true;
-      type = types.bool;
-      description = ''
-        Whether to enable cgroup accounting; see {manpage}`cgroups(7)`.
-      '';
-    };
-
-    extraConfig = mkOption {
-      default = "";
-      type = types.lines;
-      example = "DefaultLimitCORE=infinity";
+    settings.Manager = mkOption {
+      default = { };
+      type = lib.types.submodule {
+        freeformType = types.attrsOf unitOption;
+      };
+      example = {
+        DefaultLimitCORE = "infinity";
+      };
```

Comment:
Let’s not have a useless example. I’m fine just dropping it if we can’t think of something else to put in there. (`CtrlAltDelBurstAction = "none";`? The watchdog stuff?)